fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml

```diff
@@ -71,4 +71,5 @@
     <include file="parts/0049_add_send_asynchronous_events_job.xml" relativeToChangelogFile="true"/>
     <include file="parts/0050_add_reverse_flag_disbursement_details.xml" relativeToChangelogFile="true"/>
     <include file="parts/0051_external_event_table_category_info.xml" relativeToChangelogFile="true"/>
+    <include file="parts/0052_loan_transaction_chargeback.xml" relativeToChangelogFile="true"/>
```

Comment:
@josehernandezfintecheandomx Wrong file name!
Also i dont think this change is for this PR!